spec/support/session_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module Features
+  module SessionHelpers
+    def sign_in(user)
```

Comment:
Extracted this out into support helper. I would say we should name this differently? Like `login_as` or something different as the devise helpers (which are only being loaded currently in the controller/helper tests) also use `sign_in` name